web-dashboard/src/pages/guild/Logging.tsx

```
@@ -0,0 +1,77 @@
+              return (
+                <div key={event.label} className="flex items-center justify-between rounded-lg border border-border p-4">
+                  <div className="flex items-center gap-3">
+                    <div className="rounded-lg bg-primary/10 p-2">
+                      <Icon className="h-5 w-5 text-primary" />
+                    </div>
+                    <div>
+                      <div className="font-medium">{event.label}</div>
+                      <div className="text-sm text-muted-foreground">{event.description}</div>
+                    </div>
+                  </div>
+                  <button className="relative inline-flex h-6 w-11 items-center rounded-full bg-muted">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)